release-version

```diff
@@ -1,3 +1,3 @@
 # This file specifies the TiDB Dashboard internal version, which will be printed in `--version`
 # and UI. In release branch, changing this file will result in publishing a new version and tag.
-2022.12.01.1
+v6.5.0
```

Comment:
why we change the version/tag naming rule?

Reply:
Start from this TiDB release, the tidb dashboard will exist as an optional independent component. So from now on, each tidb release must have a corresponding dashboard release.